### PR TITLE
auto launch for windows.

### PR DESCRIPTION
**Pull request by rbarros-daitan**
Code implements ELECTRON-25 which describes AutoLaunch for Windows settings. Code also prepared the settings for MinimizeOnClose param.

### CODE REVIEW

**Comment by lneir** [comment]
please provide a description above for this commit.

File: config/Symphony.config [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
     "url": "https://foundation-dev.symphony.com"
+    "launchOnStartup": true,
+    "minimizeOnClose": true
```
Comment (lneir) [inline config/Symphony.config]: why is minimize on close code here, that is a separate ticket?

File: js/getConfig.js [hunk 2]
```diff
@@ -50,6 +51,14 @@ var getConfig = function () {
                 }).catch(function (){
                     resolve(config);
                 });
+
+                if (config.launchOnStartup === "true"){
```
Comment (lneir) [inline js/getConfig.js]: does this handle both bool and string value?

File: js/getConfig.js [hunk 3]
```diff
@@ -50,6 +51,14 @@ var getConfig = function () {
...
+                    const execFile = 'Symphony.exe';
+                    let execPath = path.join(app.getAppPath(), execFile);
+                    ws.create("%APPDATA%/Microsoft/Windows/Start Menu/Programs/Startup/Symphony.lnk", execPath);
```
Comment (lneir) [inline js/getConfig.js]: does this work for both install for all and install for individual user?

File: js/menus/menuTemplate.js [hunk 4]
```diff
@@ -156,9 +170,68 @@ function getTemplate(app) {
...
+    {
+      // Window menu when Windows.
+        template[2].submenu.push(
+            {
+                label: 'Auto Launch on Windows Startup', 
+                type: 'checkbox', 
+                checked: launchOnStartup,
+                click: function (item) {
+                    if (item.checked){
+                        const execFile = 'Symphony.exe';
```
Comment (lneir) [inline js/menus/menuTemplate.js]: this looks like it is overwrite global value.  if user changes, it should only effect their user setting.  that is, if user has not set a value, then user global value, if user has modified then use the user/individual setting.

File: config/Symphony.config (hunk 1 above)
Comment (rbarros-daitan) [inline config/Symphony.config]: As the code to load/save minimizeOnClose settings is almost the same of auto launch, I have already prepared it together so ELECTRON-40 has only to implement the menu behavior itself.

**Pull request closed without merging**